src/client/js/partials/breaches.js

```diff
@@ -80,11 +85,10 @@ async function updateBreachStatus (input) {
 }
 
 function renderResolvedCounts () {
-  const nrResolved = breachesPartial.querySelectorAll(`[data-status='resolved'][data-email='${state.selectedEmail}']`).length
-  const nrUnresolved = breachesPartial.querySelectorAll(`[data-status='unresolved'][data-email='${state.selectedEmail}']`).length
-  resolvedCountOutput.textContent = nrResolved
-  unresolvedCountOutput.textContent = nrUnresolved
-  statusFilter.toggleAttribute('hidden', nrResolved === 0 && nrUnresolved === 0)
+  state.resolvedCount = breachesPartial.querySelectorAll(`[data-status='resolved'][data-email='${state.selectedEmail}']`).length
```

Comment:
Nice, I always struggle with `nrOf` names, but `Count` is a good suffix to use :clap: